### ISSUE TITLE
Missing keywords

### ISSUE THREAD

**Issue by tami5** [clojure-lsp/clojure-lsp#649]
**Describe the bug**

I was expecting this to be fixed in #630. But today I played around with xtdb and no namespace-qualifed keys at all 😟 :

![Screen Shot 2021-11-23 at 11 18 46 PM](https://user-images.githubusercontent.com/65782666/143097944-185c1617-bcf2-4db8-87c2-99df07c0500f.png)

**To Reproduce**
```clojure
(ns test-ns
  (:require [xtdb.api :as xt]))

(::xt/|)
```
**Expected behavior**
![Screen Shot 2021-11-23 at 11 17 53 PM](https://user-images.githubusercontent.com/65782666/143097879-4c8beda8-28e3-41b2-bcdc-acb8edb635c0.png)

**User details**
 - OS: MacOS
 - Editor: nvim
 - Version:
    - clojure-lsp 2021.11.16-16.52.14
    - clj-kondo 2021.10.20-SNAPSHOT

**Comment by ericdallo** [comment]
Yeah, I confirmed the issue, I think we left a corner case with aliased keys, I'll double-check it, thanks for the repro

**Comment by ericdallo** [comment]
@tami5 made a commit on master partially fixing it, now we show the keywords completions that are `:reg`, keywords that are defined like via `s/def`.
The only thing is not working yet is completion with end slash like `::xt/`, I'll try to fix it soon

**Comment by tami5** [comment]
Thanks @ericdallo, I've tried current version but seems to be still an issue. maybe my editor 😆 , still the same behavior.

**Comment by ericdallo** [comment]
Did you try the latest master commit?

**Comment by tami5** [comment]
Yes, #4f43a3f1, ran just make in clojure-lsp directory

**Comment by ericdallo** [comment]
Quite odd 🤔 

About the completion on empty slash, we need to do some changes on clj-kondo I think: https://clojurians.slack.com/archives/CHY97NXE2/p1637792912377700

**Comment by ericdallo** [comment]
> The only thing is not working yet is completion with end slash like ::xt/, I'll try to fix it soon

@tami5 I made one more commit with a workaround for this

Everything from this issue is working for me now, it's odd that it doesn't for you

**Comment by tami5** [comment]
Oh wait there is something. It's a bit weird behavior  

https://user-images.githubusercontent.com/65782666/143330024-03466c79-7440-4967-b08b-c9cb70f8a4e9.mov

For comparison here's how it's behaving with conjure

https://user-images.githubusercontent.com/65782666/143330194-16d04627-caff-4946-8ae6-a5ec95b0ebf6.mov


Now I'm shocked how in consistent the results comparing to last time I tried it.

**Comment by ericdallo** [comment]
that's really weird indeed, not sure is something that client does though :(
